test/algorithms/ground_state_solvers/test_adapt_vqe.py

```diff
@@ -60,6 +65,18 @@ def setUp(self):
         self.expected = -1.85727503
 
         self.qubit_converter = QubitConverter(ParityMapper())
+       
```

Comment:
```suggestion

```
You cannot have trailing whitespace, otherwise the linter will complain.
I am not sure if this is the only reason why the style checker is also failing. I suggest that you run `make black` locally to ensure that is fixed.
You can also run `make lint` locally to check that the linter passes.